hathor/indexes/manager.py

```diff
@@ -35,13 +38,21 @@
 logger = get_logger()
 
 
+class _IndexFilter(Enum):
```

Comment:
Why don't you move it to a class attribute for each index?

Reply:
Mainly because of tips-index and timestamp-index. The interface can't be a classmethod/classattribute because we use different instances of tips-index/timestamp-index with different intended filtering. The interface can be a method/attribute, where tips-index/timestamp-index could have a parameter to store their preference, but I avoided adding this change to those indexes, currently they don't know if they are being fed all txs, only blocks or only txs. I think the way we use tips-index/timestamp-index, even if not straightforward, is consistent, if we _partially_ add the knowledge about what types of transactions they are being fed to inside the implementation, it will significantly confusing IMO.

I'd rather either _fully_ move that knowledge to the index (as in, the manager will not give any special treatment to these indexes), or not move them at all, and let the manager continue being the only place where that knowledge is. I opted for the latter because it's less changes overall, but I do think that an ideal indexes-manager would not know the specifics about the indexes, but that can be refactored later.

Reply:
Got it. I feel we should move these decisions to the indexes and let the manager just coordinate them. I guess we can do it later but still on the speed up initialization project. Would you create an issue to get this done, please?

Reply:
OK

Reply:
Done: #418

Reply:
Thanks!